frontend_tests/node_tests/stream_data.js

```diff
@@ -538,11 +538,18 @@ zrequire('stream_data');
     var desc = {
         name: 'no_desc',
         stream_id: 1002,
-        description: '<p>rendered desc</p>'
+        description: '<p>rendered desc</p>',
```

Comment:
This should be with the `node test: Cover stream_data.render_stream_description.` commit.